libs/lib-client/contrib-installer/src/lib/contrib-installer.component.ts

```diff
@@ -51,7 +51,13 @@ export class FlogoInstallerComponent {
 
   closeModal() {
     console.log('Close Modal.');
-    this.control.close();
+    this.status = 'cancel';
```

Comment:
what's the difference between `FlogoInstallerComponent.status` vs `FlogoInstallerComponent._status`? also, does it have to be a class property or can we get away with making status a local variable? i.e. `const status = 'cancel'`